tests/spec/core/inlines-spec.js

```diff
@@ -367,4 +367,31 @@ describe("Core - Inlines", () => {
       "https://infra.spec.whatwg.org/#map-iterate"
     );
   });
+
+  it("processes {{ forContext/term }} IDL", async () => {
+    const body = `
+      <section>
+        <p id="link1">{{ Window.event }}</p>
+        <p id="link2">{{ Window/event }}</p>
```

Comment:
So this is to render `event` while linking to `Window.event`, right? Personally thought Wikipedia-like `{{ Window.event | event }}` but this one looks clean.

Reply:
The Wikipedia-like also looks interesting. 
Went with `{{ Window/event }}` as we already have `[= For/link =]`. 

`{= Window/event =}` hmmm 🤔

Reply:
Do `[= Window/event =]` and `{{ Window/event }}` link to the same target?

Reply:
No. The second one is specific to IDL (and a guarantee that the terms won't be lower cased).

Reply:
Just noting that {{Foo/bar}} will match what BikeShed does also.